### ISSUE TITLE
Garbled characters when importing source code

### ISSUE THREAD

**Issue by chiepu3** [Tampermonkey/tampermonkey#1456]
(Please fill out the issue template with your details)

## Expected Behavior
In the Utilities section of Tampermonkey's settings, the source code is garbled when using either "Import from File" & "Import from URL".
This occurs in any file (UTF-8 or ANSI). I have tried changing the newline code from CRLF to LF, etc., but this did not solve the problem.

## Specifications

- Vivaldi: (5.1.2567.66)
- TM: (4.15)
- OS: (Windows 11 Pro 21H2)

## Script

(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name         scp en to jp
// @namespace    http://tampermonkey.net/
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match        https://scp-wiki.wikidot.com/scp-2639
// @icon         https://www.google.com/s2/favicons?domain=wikidot.com
// @grant        none
// ==/UserScript==

(function() {
    'use strict';
    window.onload = function(){
        var foo = location.pathname;
            var test = document.createElement("a"); //新しい要素を"タグ名"で作成し、変数test に代入
            test.href = ("http://scp-jp.wikidot.com" + foo); //リンク先をURLで指定
            var str = document.createTextNode("[日本版へ移動]"); //テキストノードを追加し変数str に代入
            test.appendChild(str); //変数test にテキストノードを追加
            test.setAttribute('target','_blank');
        test.setAttribute('id','clickme'); //変数test に target="_blank" を設定
            test.classList.add("add-link"); //変数test にクラス名"add-link"を追加
            document.getElementById("page-title").appendChild(test);
        document.getElementById("clickme").click();//#lga要素に変数test を追加
    };
})();
```

Please note that I am a Japanese speaker and use translations

**Comment by chiepu3** [comment]
![image](https://user-images.githubusercontent.com/84263077/159634951-84e25fd0-e72c-4151-8ad2-9dcab2ce72be.png)

**Comment by derjanb** [comment]
Should be fixed at 4.16.6158 (in review|[crx](https://tampermonkey.net/crx/tampermonkey_beta_current.crx)|[xpi](https://firefox.tampermonkey.net/firefox-current-beta.xpi))

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
Firefox users please install the [BETA version](https://www.tampermonkey.net/index.php?browser=firefox) or check for BETA version updates at `about:addons` 

For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
